WTWSMS/localisation/WtWSMS_settings.csv

```diff
@@ -30,4 +30,9 @@ RULE_RISE_OF_ISLAM_DISABLED_DESC;The work in progress, untested Rise of Islam fe
 RULE_RISE_OF_ISLAM_EXPERIMENTAL_DYNAMIC;Experimental Dynamic;Dynamique exp�rimentale;;;;;;;;;x
 RULE_RISE_OF_ISLAM_EXPERIMENTAL_DYNAMIC_DESC;The work in progress, untested Rise of Islam feature is enabled, in the dynamic form, starting with the birth of the Prophet. �RWARNING This is not recommended!�!;Le travail est en cours de d�veloppement, la fonctionnalit� "La Mont�e de l'Islam" est activ�e, dans sa forme dynamique, d�butant avec la naissance du Proph�te. �RAVERTISSEMENT Ce n'est pas recommand�!�!;;;;;;;;;;x
 RULE_RISE_OF_ISLAM_EXPERIMENTAL_HISTORICAL;Experimental Historical;Exp�rimentation historique;;;;;;;;;x
-RULE_RISE_OF_ISLAM_EXPERIMENTAL_HISTORICAL_DESC;The work in progress, untested Rise of Islam feature is enabled, in the historical form, starting with the Prophet declaring war. �RWARNING This is not recommended!�!;Le travail est en cours de d�veloppement, la fonctionnalit� "La Mont�e de l'Islam" est activ�e, dans sa forme historique, d�butant avec la naissance du Proph�te. �RAVERTISSEMENT Ce n'est pas recommand�!�!;;;;;;;;;;x
\ No newline at end of file
+RULE_RISE_OF_ISLAM_EXPERIMENTAL_HISTORICAL_DESC;The work in progress, untested Rise of Islam feature is enabled, in the historical form, starting with the Prophet declaring war. �RWARNING This is not recommended!�!;Le travail est en cours de d�veloppement, la fonctionnalit� "La Mont�e de l'Islam" est activ�e, dans sa forme historique, d�butant avec la naissance du Proph�te. �RAVERTISSEMENT Ce n'est pas recommand�!�!;;;;;;;;;;x
+RULE_DYNAMIC_CUSTOM_REALMS;AI Dynamic Custom Realms;;;;;;;;;;x
+RULE_DYNAMIC_CUSTOM_REALMS_UNRESTRICTED;Unrestricted;;;;;;;;;;x
+RULE_DYNAMIC_CUSTOM_REALMS_UNRESTRICTED_DESC;The AI can create Custom Realms akin to the player.;;;;;;;;;;x
+RULE_DYNAMIC_CUSTOM_REALMS_UNRESTRICTED;Restricted;;;;;;;;;;x
+RULE_DYNAMIC_CUSTOM_REALMS_UNRESTRICTED_DESC;The AI cannot create custom realms, except in extreme, extenuating circumstances. �RWARNING This is not recommended!�!;;;;;;;;;;x
```

Comment:
The only joke here is you.

I didn't have room in the Rules Menu to explain why it is not recommended.
It is not reccommended because it, along with the draconian restrictions on most de jure titles, means the AI can never advance, and will succession.

If you are rrally going to be a butt munch about it, I'll change it to "this makes the game easier".